Doc/library/http.server.rst

```diff
@@ -193,7 +195,7 @@ provides three different variants:
 
    .. method:: handle_expect_100()
 
-      When a HTTP/1.1 compliant server receives an ``Expect: 100-continue``
+      When an HTTP/1.1 conformant server receives an ``Expect: 100-continue``
```

Comment:
`compliant` was correct, why change it?

Reply:
@merwok Actually the two terms are not synonyms; ‘conformance’ is voluntary, while ‘compliance’ is mandatory:
https://www.kbis.com.au/difference-between-conformance-and-compliance/

The term ‘conformant’ is the proper term, used in the HTTP specification:
https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#protocol.version

And the HTTP specification makes an explicit distinction between ‘conformant’ and ‘compliant’ here:
https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#rfc.section.2.2.p.4):

> An implementation is considered conformant if it complies with all of the requirements associated with the roles it partakes in HTTP.

Reply:
What do you think?

Reply:
Do you still prefer ‘compliant’? If that is what is preventing this PR from being merged, we can keep ‘compliant’.

Reply:
I haven’t had time to review the latest changes + check if the rest of the documentation uses compliant and/or conformant.
There is still time before 3.11 beta!

Reply:
Yes there is still time. Should I add an entry in What’s new since it is a new feature?

Reply:
News fragment is enough!

Reply:
Alright.

Reply:
Hi @merwok. The 3.11 beta will be released next week. Could we merge?